swarm/api/client/client.go

```diff
@@ -728,3 +752,57 @@ func (c *Client) GetFeedRequest(query *feed.Query, manifestAddressOrDomain strin
 	}
 	return &metadata, nil
 }
+
+func GetClientTrace(traceMsg, metricPrefix, ruid string, tn *time.Time) *httptrace.ClientTrace {
```

Comment:
Remove `fetch` and `clienttrace` prefixes - these can be added to `metricsPrefix` if needed.